src/html5-qrcode-scanner.ts

```diff
@@ -137,8 +147,13 @@ export class Html5QrcodeScanner {
         if (!document.getElementById(elementId)) {
             throw `HTML Element with id=${elementId} not found`;
         }
-
+        
         this.currentScanType = Html5QrcodeScanType.SCAN_TYPE_CAMERA;
+        //checking if selectedScanType is file then, setting currentScanType accordingly, else no change 
+        if(this.config.selectedScanType === Html5QrcodeSelectedScanType.SELECTED_SCAN_TYPE_FILE){
```

Comment:
Nit, please lint the code

```js
if (condition) {

}
```

(note the spaces)

Reply:
Ditto elsewhere in the change. 

Also try that each line doesn't exceed 120 characters.